CHANGELOG.md

```diff
@@ -1,65 +1,86 @@
-Changelog
-=========
+# Changelog
+
+### 1.0.1
+
+- Adding support to per Hour limit
```

Comment:
Please change to: 
 - Adds support for the hourly query limit on the Account API endpoint
 - Fixes an issue with handling of proxies (thanks @neok0!)

Reply:
Roger that!